assets/css/extra.css

```
@@ -1,290 +1,529 @@
+	transform: translateY(0);
+	transition:
+		opacity 0.3s,
+		transform 0.3s;
+	animation: slideUp 0.3s ease-out;
 }
-#selectLocation:active {
-	border-bottom: none;
-	top: 4px;
-	margin-bottom: 4px;
+
+@keyframes slideUp {
```

Stylelint (17.14.0)

[error] Expected keyframe name "slideUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)